diamm/serializers/search/source.py

```diff
@@ -140,3 +141,10 @@ def get_public_images_b(self, obj):
             return True
         else:
             return False
+
+    def get_geographic_area_ii(self, obj):
+        geographic_area = list()
```

Comment:
This can be done much simpler.

`geographic_area = obj.provenance.values_list('city_id', 'country_id', 'protectorate_id', 'region_id')`

However, I'm unsure what you're going to do with these IDs. You will need to get their names in order to provide links to them, right?